astropy/wcs/tests/test_utils.py

```diff
@@ -353,6 +368,52 @@ def test_wcs_to_celestial_frame():
     assert isinstance(frame, Galactic)
 
 
+def test_wcs_to_body_frame():
+    # Import astropy.coordinates here to avoid circular imports
```

Comment:
I don't see how you can get a circular import in a test function; just put it on top!